src/index.js

```diff
@@ -87,6 +87,7 @@ async function run(request, ctx) {
     }
   }
   const html = await res.text();
+  const imgSrcPolicy = (res.headers.get('x-html2md-img-src') || 'self').split(' ');
```

Comment:
more tolerant split:

```suggestion
  const imgSrcPolicy = (res.headers.get('x-html2md-img-src') || 'self').split(/\s+/);
```